components/molecules/ContributionsTreemap/contributions-treemap.tsx

```diff
@@ -8,7 +8,8 @@ import type { NodeMouseEventHandler, NodeProps } from "@nivo/treemap";
 
 interface ContributionsTreemapProps {
   data: any;
-  color: string;
+  // loose typing this as i can't find the import for  OrdinalColorScaleConfig<ComputedNodeWithoutStyles<object>> | undefined
+  color: any;
```

Comment:
Refactor (blocking): The type can be pulled from the Treemap props, and then import these

```diff
- import type { NodeMouseEventHandler, NodeProps } from "@nivo/treemap";
+ import type { NodeMouseEventHandler, NodeProps, TreeMapCommonProps } from "@nivo/treemap";
+ import { Datum } from "@nivo/line";
```

```suggestion
  color: TreeMapCommonProps<Datum>["colors"];
```

Reply:
Thanks 🍕